src/main/java/org/zone/commands/structure/misc/Messages.java

```diff
@@ -0,0 +1,304 @@
+package org.zone.commands.structure.misc;
+
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.NamedTextColor;
+import net.kyori.adventure.text.format.TextDecoration;
+import org.spongepowered.api.entity.living.player.User;
+import org.spongepowered.configurate.ConfigurateException;
+import org.zone.ZonePlugin;
+import org.zone.commands.system.ArgumentCommand;
+import org.zone.commands.system.context.ErrorContext;
+import org.zone.region.Zone;
+import org.zone.region.flag.FlagType;
+import org.zone.region.flag.move.player.leaving.LeavingFlag;
+import org.zone.region.group.Group;
+
+import java.math.BigDecimal;
+import java.util.Collection;
+import java.util.UUID;
+
+/**
+ * Store house for all messages in this plugin {@link ZonePlugin} to make translation easier
+ */
+public final class Messages {
+
+    private Messages() {
+        throw new RuntimeException("Could not construct class");
+    }
+    //Universal Messages
+    public static Component getUniversalPlayerOnlyCommandError() {
+        return Component.text("Player only command").color(NamedTextColor.RED);
+    }
+
+    public static Component getUniversalZoneSavingError(ConfigurateException ce) {
+        return Component.text("Error when saving: " + ce.getMessage()).color(NamedTextColor.RED);
+    }
+
+    public static Component getUniversalZoneRegionBuilderEnabled() {
+        return Component
+                .text("Region builder mode enabled. Run ")
+                .append(Component
+                                .text("'/zone create end'")
+                                .color(NamedTextColor.AQUA));
+    }
+
+    public static Component getUniversalDuplicateNameError() {
+        return Component
+                .text("Cannot use that name")
+                .color(NamedTextColor.RED);
+    }
+
+    public static Component getUniversalMessageTag() {
+        return Component.text("Message: ");
+    }
+
+    public static Component getUniversalNoMessageSet() {
+        return getUniversalMessageTag().append(Component.text("No message set by user")).color(NamedTextColor.RED);
+    }
+
+    public static Component getUniversalFlagMessageView(Component message) {
+        return getUniversalMessageTag().append(message).color(NamedTextColor.AQUA);
```

Comment:
No need to set the colour on this one as both the inputs are components that bring in there own colours

Reply:
ok

Reply:
yes not gave enough attention

Reply:
@mosemister